scripts/deploy.sh

```diff
@@ -12,6 +12,15 @@ source /opt/manager-vars.sh
 /opt/configuration/scripts/set-manager-version.sh $MANAGER_VERSION
 /opt/configuration/scripts/set-openstack-version.sh $OPENSTACK_VERSION
```

Comment:
Move set-ceph/set-openstack into an else block below? Avoids unnecessary change and confusion